rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -1250,18 +1251,23 @@ def run_aw2f_manifest(self, feedback_id):
 
     def run_aw3_manifest(self, job, manifest_type, genome_type):
         server_config = self.get_server_config()
-
-        # Run the AW3 Workflow
-        with GenomicJobController(job_id=job,
-                                  max_num=4000,
-                                  bq_project_id=self.gcp_env.project) as controller:
+        with GenomicJobController(job,
+                                  server_config=server_config,
+                                  max_num=4000) as controller:
             controller.bucket_name = server_config[config.DRC_BROAD_BUCKET_NAME][0]
-
             controller.generate_manifest(
-                manifest_type=manifest_type,
+                manifest_type,
                 genome_type=genome_type,
+                pipeline_id="dragen_3.7.8"
```

Comment:
:nit Can we update this to `config.GENOMIC_UPDATED_WGS_DRAGEN`

Reply:
Yep, done.